TestFoundation/TestCodable.swift

```diff
@@ -254,19 +258,31 @@ class TestCodable : XCTestCase {
     // MARK: - CGRect
     lazy var cgrectValues: [CGRect] = [
         CGRect(),
+        CGRect.zero,
         CGRect(origin: CGPoint(x: 10, y: 20), size: CGSize(width: 30, height: 40)),
         CGRect(origin: CGPoint(x: -10, y: -20), size: CGSize(width: -30, height: -40)),
         // Disabled due to limit on magnitude in JSON. See SR-5346
-        // CGRect(origin: CGPoint(x: -.greatestFiniteMagnitude / 2,
-        //                        y: -.greatestFiniteMagnitude / 2),
-        //        size: CGSize(width: .greatestFiniteMagnitude,
-        //                     height: .greatestFiniteMagnitude)),
+        // CGRect.infinite
     ]
     
     func test_CGRect_JSON() {
         for rect in cgrectValues {
             expectRoundTripEqualityThroughJSON(for: rect)
         }
+        
+        do {
```

Comment:
`do {}` is not necessary and misleading. I'll remove it. 👍 

As for `CGRect.null` in `cgrectValues`, it requires `JSONEncoder` and `JSONDecoder` configured differently than for the other test values.
Also I don't think the other test values should share the configuration that is required by `CGRect.null`
So I test it separately from the rest.

Is this approach unreasonable? Would like to hear your opinion on that.